### PR TITLE
Remove unused InvalidSignatureLength

### PR DESCRIPTION
**Pull request by dnkolegov**
This PR fixes https://github.com/consensus-shipyard/ipc-solidity-actors/issues/364

### CODE REVIEW

File: test/integration/SubnetActorDiamond.t.sol [hunk 1]
```diff
@@ -511,7 +511,7 @@ contract SubnetActorDiamondTest is Test, IntegrationTestBase {
             saManager.join{value: 10}(pubKeys[i]);
         }
 
-        vm.expectRevert(abi.encodeWithSelector(InvalidSignatureErr.selector, 4));
```
Comment (raulk) [inline test/integration/SubnetActorDiamond.t.sol:514]: @dnkolegov is it not possible to refer to the enum uint value in Solidity? 🤔

Comment (dnkolegov) [reply test/integration/SubnetActorDiamond.t.sol:506]: Fixed and added missing tests

**Comment by snissn** [comment]
thanks! love all the tests and agree with removing the enum. Putting my approval on this, but also glad that @raulk is chiming in with feedback and may have more nits or blocks

**Review by snissn — APPROVED** [review]
lgtm

**Pull request merged**